Dockerfile

```
@@ -189,9 +202,9 @@ ARG BUILD_DATE
 ARG VCS_REF
 ARG VERSION
 LABEL org.label-schema.build-date=$BUILD_DATE \
-      org.label-schema.name="FMRIPREP" \
-      org.label-schema.description="FMRIPREP - robust fMRI preprocessing tool" \
-      org.label-schema.url="http://fmriprep.org" \
+      org.label-schema.name="fMRIPrep" \
+      org.label-schema.description="fMRIPrep - robust fMRI preprocessing tool" \
+      org.label-schema.url="http://fMRIPrep.org" \
```

Just aesthetically, I don't like caps in URLs. You can take this or leave it though.

```suggestion
      org.label-schema.url="http://fmriprep.org" \
```